src/llvm-alloc-opt.cpp

```diff
@@ -44,7 +44,7 @@ static void removeGCPreserve(CallInst *call, Instruction *val)
 {
     auto replace = Constant::getNullValue(val->getType());
     call->replaceUsesOfWith(val, replace);
-    for (auto &arg: call->arg_operands()) {
+    for (auto &arg: call->args()) {
```

Comment:
This change now strips off the trailing operands. I suppose there probably weren't any, but was that intentional?

Reply:
Huh? `args()` and `arg_operands()` were implemented the same in LLVM 13 – am I missing something?

Reply:
Ah, I see they are just a simple rename many versions ago. I was accidentally looking at the FuncletPadInst instead.